src/client/Microsoft.Identity.Client/Internal/Requests/Silent/BrokerSilentStrategy.cs

```diff
@@ -79,7 +79,9 @@ private async Task<MsalTokenResponse> SendTokenRequestToBrokerAsync()
                 _logger.Info(
                     LogMessages.ErrorReturnedInBrokerResponse(msalTokenResponse.Error));
 
-                if (msalTokenResponse.Error == BrokerResponseConst.NoTokenFound)
+                if (msalTokenResponse.Error == BrokerResponseConst.NoTokenFound ||
```

Comment:
Could you prefix these constants with "Android"?

Reply:
done.

---
In reply to: [526108373](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2200#discussion_r526108373) [](ancestors = 526108373)